jenkins/pipelines/build-linux-amd64-enterprise.groovy

```diff
@@ -66,7 +66,7 @@ try {
                     }
                 }
                 dir("go/src/github.com/pingcap/enterprise-plugin") {
-                    checkout changelog: false, poll: true, scm: [$class: 'GitSCM', branches: [[name: "release-4.0"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PruneStaleBranch'], [$class: 'CleanBeforeCheckout'], [$class: 'CloneOption', timeout: 2]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', refspec: '+refs/heads/*:refs/remotes/origin/*', url: 'git@github.com:pingcap/enterprise-plugin.git']]]
+                    checkout changelog: false, poll: true, scm: [$class: 'GitSCM', branches: [[name: "release-5.0"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PruneStaleBranch'], [$class: 'CleanBeforeCheckout'], [$class: 'CloneOption', timeout: 2]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', refspec: '+refs/heads/*:refs/remotes/origin/*', url: 'git@github.com:pingcap/enterprise-plugin.git']]]
```

Comment:
Why change it directly instead of creating a new one?

Reply:
will refine later,

Reply:
https://github.com/PingCAP-QE/ci/issues/116